components/expenses/ExpenseForm.js

```diff
@@ -609,7 +624,9 @@ const ExpenseFormBody = ({
                     onClick={() => setShowResetModal(true)}
                   >
                     <Undo size={11} />
-                    <Span mx={1}>{formatMessage(msg.resetExpense)}</Span>
+                    <Span mx={1}>
+                      {editingExpense ? formatMessage(msg.cancelEditExpense) : formatMessage(msg.clearExpenseForm)}
```

Comment:
:ice_cream: Personal preference that simplifies this kind of code

```suggestion
                      {formatMessage(editingExpense ? msg.cancelEditExpense : msg.clearExpenseForm)}
```